src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Manage/DisplayStatistics.cshtml

```diff
@@ -24,12 +24,12 @@
     <tbody>
         @foreach (var currentStat in Model)
         {
-            <tr>
-                <th scope="row" class="col-xs-1"></th>
-                <td style="text-align: center;">@currentStat.TestNumber</td>
-                <td style="text-align: center;">@currentStat.CorrectAnswerDate</td>
-                <td style="text-align: center;">@currentStat.WrongAnswerCount</td>
-            </tr>
+        <tr>
+            <th scope="row" class="col-xs-1"></th>
+            <td style="text-align: center;">@currentStat.TestNumber</td>
+            <td style="text-align: center;">@currentStat.CorrectAnswerDate</td>
+            <td style="text-align: center;">@foreach (var WrongAnswer in currentStat.WrongAnswers)@WrongAnswer
```

Comment:
I suggest adding a column with the wrong answers count, and then the column with string WrongAnswers

Reply:
Like this?
![Screenshot_1](https://user-images.githubusercontent.com/44707515/107533166-f0a16780-6bbe-11eb-86f2-d4d2b4a4ea7f.png)

Reply:
yes